Source/WPEFramework/Controller.cpp

```diff
@@ -70,6 +71,9 @@ namespace Plugin {
 
         _resumes.clear();
         _service = service;
+        
+        Connector::Connector::Instance().Announce(service);
```

Comment:
Use the C function i.s.o. the C++ functionality, so remark for this class declaration.